feature-one.js

```diff
@@ -1,4 +1,5 @@
 console.log("This has no feature yet.")
 
-function friendRequest();
-    answer = window.prompt('Will you be my friend?');
\ No newline at end of file
+function friendRequest() {
+    answer = window.prompt('Will you be my friend?');
```

Comment:
make sure to declare the answer variable with "var" or "let"

Reply:
Oh cool Thank you. Got it!